chart/k8gb/templates/service-delete-hook/job.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: hook-service-delete
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-upgrade
+    "helm.sh/hook-delete-policy":  hook-succeeded
+spec:
+  template:
+    metadata:
+      name: hook-service-delete
+    spec:
+      restartPolicy: Never
+      serviceAccountName: hook-sa
+      automountServiceAccountToken: true
+      containers:
+        - name: kubectl
+          image: {{ .Values.k8gb.hookImageRepo }}:{{ .Values.k8gb.hookImageTag }}
+          args:
+            - delete
+            - svc
+            - -n
+            - {{ .Release.Namespace }}
+            - k8gb-coredns
```

Comment:
What if it doesn't exist? What is the new coredns service name structure? Is it the same or auto-generated by the upstream chart?

Reply:
Hmm. You're right, it should be coredns.service.serviceName

Reply:
Will delete by label instead